### PR TITLE
updated pedaitrics.csv

### PR DESCRIPTION
**Pull request by Incei**
converted units for ratnapalan et al, and corrected units nyberg et al.

### CODE REVIEW

File: Pediatrics.csv [hunk 1]
```diff
@@ -256,23 +256,23 @@ ID,Study,Reference,Grouping,Analyte,t0,tend,t Unit,AUC Avg,AUC AvgUnit,AUC AvgTy
...
+5217,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID6-DDI 8m,Digoxin,0,24,h,,,,,,,,,,,,,,,18.879,mL/min,Individual,,,,,Clearance converted from mL/min/m2 to mL/min using reported m2 values
+5218,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID7-8m,Digoxin,0,24,h,,,,,,,,,,,,,,,46.48,mL/min,Individual,,,,,Clearance converted from mL/min/m2 to mL/min using reported m2 values
+5219,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID7-DDI 9m,Digoxin,0,24,h,,,,,,,,,,,,,,,21.875,mL/min,Individual,,,,,Clearance converted from mL/min/m2 to mL/min using reported m2 values
+5220,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID8-DDI 7m,Digoxin,0,24,h,,,,,,,,,,,,,,,62.028,mL/min,Individual,,,,,Clearance converted from mL/min/m2 to mL/min using reported m2 values
+5221,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID8-DDI 8m,Digoxin,0,24,h,,,,,,,,,,,,,,,35.532,mL/min,Individual,,,,,Clearance converted from mL/min/m2 to mL/min using reported m2 values
```
[the remark this answers is not in the document]
Comment (Incei) [reply Pediatrics.csv:276]: no it means months. it is their age.

[the remark this answers is not in the document]
Comment (Incei) [reply Pediatrics.csv:276]: Let's wait, eitherway, the R-Quali runner does not like 2 different units in the table.. :(. Previously i fixed it by adding the row:

  #' Run the `QualificatitonWorklfow`
  workflow$plotPKRatio$settings$units$CL <- "mL/min/kg"

However, when having a second unit in the observed dataset, it gives again the error...

rror in rClr::clrCall(dimensionTask, "ConvertToUnit", dimension, targetUnit,  : 
  Type:    OSPSuite.Utility.Exceptions.OSPSuiteException
Message: Unit 'mL/min' is not defined in dimension 'Flow per weight'.

[the remark this answers is not in the document]
Comment (Incei) [reply Pediatrics.csv:276]: https://github.com/Open-Systems-Pharmacology/QualificationRunner/issues/126

**Review by Yuri05 — CHANGES_REQUESTED** [review]
CL units must be given per body weight. Other units are not accepted
grafik

**Comment by Yuri05** [comment]
Pediatrics.csv is not saved as UTF-8

File: Pediatrics.csv [hunk 2]
```diff
@@ -256,23 +256,23 @@ ID,Study,Reference,Grouping,Analyte,t0,tend,t Unit,AUC Avg,AUC AvgUnit,AUC AvgTy
...
-5218,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID15,Digoxin,0,24,h,,,,,,,,,,,,,,,132.8,mL/min/m2,Individual,,,,,
-5219,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID16,Digoxin,0,24,h,,,,,,,,,,,,,,,62.5,mL/min/m2,Individual,,,,,
-5220,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID17,Digoxin,0,24,h,,,,,,,,,,,,,,,172.3,mL/min/m2,Individual,,,,,
-5221,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID18,Digoxin,0,24,h,,,,,,,,,,,,,,,98.7,mL/min/m2,Individual,,,,,
-5222,Nyberg 1980,https://www.ncbi.nlm.nih.gov/pubmed/7398751,neonates,Digoxin,0,24,h,,,,,,,,,,,,,,,3,mL/min/1.73m2,arith. Mean,,,,,Taken from Nyber 1978 reported doses and serum concentrations
-5223,Nyberg 1980,https://www.ncbi.nlm.nih.gov/pubmed/7398751,infants (moderate cl),Digoxin,0,24,h,,,,,,,,,,,,,,,4.5,mL/min/1.73m2,arith. Mean,,,,,Taken from Nyber 1978 reported doses and serum concentrations
+5204,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID1-1y,Digoxin,0,24,h,,,,,,,,,,,,,,,3.9072,mL/min/kg,Individual,,,,,Clearance converted from mL/min/kg to mL/min using reported m2 values and PK-sim generated BW according to the given age
```
Comment (Yuri05) [inline Pediatrics.csv:259]: the comments should be "...converted from mL/min**/m2** to mL/min**/kg**..." right?

File: Pediatrics.csv [hunk 3]
```diff
@@ -256,23 +256,23 @@ ID,Study,Reference,Grouping,Analyte,t0,tend,t Unit,AUC Avg,AUC AvgUnit,AUC AvgTy
...
+5219,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID7-DDI 9m,Digoxin,0,24,h,,,,,,,,,,,,,,,2.610381862,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5220,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID8-DDI 7m,Digoxin,0,24,h,,,,,,,,,,,,,,,8.508641975,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5221,Ratnapalan 2003,https://www.ncbi.nlm.nih.gov/pubmed/12756393,ID8-DDI 8m,Digoxin,0,24,h,,,,,,,,,,,,,,,4.537931034,mL/min/kg,Individual,,,,,Clearance converted from mL/min/m2 to mL/min/kg using reported m2 values and PK-sim generated BW according to the given age
+5222,Nyberg 1980,https://www.ncbi.nlm.nih.gov/pubmed/7398751,neonates,Digoxin,0,24,h,,,,,,,,,,,,,,,3,mL/min/kg,arith. Mean,,,,,Taken from Nyber 1978 reported doses and serum concentrations
+5223,Nyberg 1980,https://www.ncbi.nlm.nih.gov/pubmed/7398751,infants (moderate cl),Digoxin,0,24,h,,,,,,,,,,,,,,,4.5,mL/min/kg,arith. Mean,,,,,Taken from Nyber 1978 reported doses and serum concentrations
```
Comment (Yuri05) [inline Pediatrics.csv:278]: The values for Nyberg in `ml/min/kg `are the same as before in `mL/min/1.73m2` ?

Comment (Incei) [reply Pediatrics.csv:278]: correct, it was a copy error.

Comment (Yuri05) [reply Pediatrics.csv:278]: ok, after this was fixed, we can merge

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**